python/llm/test/benchmark/csv_to_html.py

```diff
@@ -30,8 +30,23 @@ def highlight_vals(val, max=3.0):
     else:
         return ''
 
-def is_diffs_within_normal_range(diff1, diff2, threshold=5.0):
-    return not any(diff < (-threshold) for diff in diff1 + diff2)
+def is_diffs_within_normal_range(diff1, diff2, best_diff1, best_diff2, threshold=5.0):
```

Comment:
Has discussed with @liu-shaojun , there is no need to add best_diff1 and best_diff2 into this function to judge whether they are in the normal range :)

Reply:
Besides, in the main branch
![image](https://github.com/intel-analytics/BigDL/assets/54161268/db72f0ff-88c9-493f-b9c7-f87554e2ec10)
This error has been fixed. Please rebase this PR. Thank you :)

Reply:
Sure.